src/schemas/execute.yaml

```diff
@@ -0,0 +1,369 @@
+  description: The error messages are suggestions, and clients might implement different error messages. However, the error codes are enforced by the spec.
+  type: object
+  required:
+    - status
+    - returnData
+    - gasUsed
+    - error
+  properties:
+    status:
+      title: Call Status Failure
+      type: string
+      pattern: ^0x0$
+    returnData:
+      title: Return data
+      $ref: '#/components/schemas/bytes'
+    gasUsed:
+      title: Return gasUsed
+      $ref: '#/components/schemas/uint64'
+    error:
+      oneOf:
+        - code: -32000
+          message: 'Execution reverted'
+        - code: -32015
+          message: 'VM execution error'
+CallResultSuccess:
+  title: Result of call success
+  type: object
+  required:
+    - status
+    - returnData
+    - gasUsed
+    - logs
+  properties:
+    status:
+      title: Call Status Success
+      type: string
+      pattern: ^0x1$
+    returnData:
+      title: Return data
+      $ref: '#/components/schemas/bytes'
+    gasUsed:
+      title: Return gasUsed
+      $ref: '#/components/schemas/uint64'
+    logs:
+      title: Return logs
+      type: array
+      items:
+        $ref: '#/components/schemas/CallResultLog'
+CallResultInvalid:
+  title: Result of call not being valid (nonce, baseFeePerGas, etc)
+  description: The error messages are suggestions, and clients might implement different error messages. However, the error codes are enforced by the spec.
+  type: object
+  required:
+    - status
+    - error
+  properties:
+    status:
+      title: Call Status Invalid
+      type: string
+      pattern: ^0x2$
+    error:
+      oneOf:
+        - code: -32000
+          message: Missing or invalid parameters
+        - code: -32005
+          message: Limit exceeded
+        - code: -32015
+          messagE: Execution error
+        - code: -32016
+          message: Timeout
+        - code: -32602
+          message: Invalid params
+        - code: -32603
+          message: The Ethereum node encountered an internal error
+        - code: -38010
+          message: Transactions nonce is too low
+        - code: -38011
+          message: Transactions nonce is too high
+        - code: -38012
+          message: Transactions baseFeePerGas is too low
+        - code: -38013
+          message: Not enough gas provided to pay for intrinsic gas for a transaction
+        - code: -38014
+          message: Insufficient funds to pay for gas fees and value fo a transaction
+        - code: -38015
+          message: Block gas limit exceeded by the block's transactions
+        - code: -38020
+          message: Block number in sequence did not increase
+        - code: -38021
+          message: Block timestamp in sequence did not increase
+        - code: -38022
+          message: MovePrecompileToAddress referenced itself in replacement
+        - code: -38023
+          message: Multiple MovePrecompileToAddress referencing the same address to replace
```

Comment:
Add:

- Sender is not an EoA which is checked if validation: true
- Max init code size exceeded

Question:

- Should we return invalid params error in case priority fee is more than maxFeePerGas?
- What if maxFeePerGas is lower than the block's baseFeePerGas? Is this what is meant by `transactions baseFeePerGas is too low`?
- Do we need both "missing and invalid params" and "invalid params"?
- What is "limit exceeded", what limit?
- I believe "block number/timestamp" not incremented and `MovePrecompileToAddress` errors should be top-level. We should immediately quit and error when seeing those.

Reply:
I'll add those errors:

Questions:
>Should we return invalid params error in case priority fee is more than maxFeePerGas?
I think so, as thats invalid transaction

>What if maxFeePerGas is lower than the block's baseFeePerGas? Is this what is meant by transactions baseFeePerGas is too low?
Yeah that error is for that, do you have ideas on how to name it better?

>Do we need both "missing and invalid params" and "invalid params"?
Probably not, just invalid params is enough

>What is "limit exceeded", what limit?
This is for errors when the datatype cannot fit the data, eg, as gaslimit is `uint64` and you try to give it number that is bigger than uint64

>I believe "block number/timestamp" not incremented and MovePrecompileToAddress errors should be top-level. We should immediately quit and error when seeing those.
That is top level error, only Execution reverted and VM execution error errors are not top level

Reply:
ah sorry, the `Limit exceeded` error is for when you spend more gas than the total gas limit is set by the client. I think this could be clarified. What error eth_call throws in that case?